inst/templates/github_ci/dot.github/workflows/rhino-test.yml

```
@@ -7,11 +7,15 @@ jobs:
     name: Run linters and tests
     runs-on: ubuntu-20.04
     env:
-      R_VERSION: '4.1.0'
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
     steps:
       - name: Checkout repo
         uses: actions/checkout@v2
+        
+      - name: Extract R.version from renv.lock
+        run: |
+          r_version=`jq .R.Version renv.lock | xargs`
+          echo "R_VERSION=$r_version" >> $GITHUB_ENV
```

Consider this simplified version. Also please configure your editor to strip trailing whitespace automatically when saving :wink:
```suggestion
        
      - name: Extract R version from lockfile
        run: printf 'R_VERSION=%s\n' "$(jq --raw-output .R.Version renv.lock)" >> $GITHUB_ENV
```